src/backend/dx12/shaders/blit.hlsl

```diff
@@ -0,0 +1,38 @@
+
+Texture2DArray blit_src : register(t0);
+SamplerState blit_sampler : register(s0);
+
+cbuffer region : register(b0) {
+    float2 src_start;
+    float2 src_end;
+    float z;
+};
+
+struct VsOutput {
+    float4 position : SV_POSITION;
+    float3 src_coords : TEXCOORD;
+};
+
+// Create a screen filling triangle
+VsOutput vs_blit_2d(uint id: SV_VertexID) {
+    VsOutput output;
+    switch (id) {
+        case 0:
+            output.position = float4(-1.0, -1.0, 0.0, 0.0);
+            output.src_coords = float3(src_start, z);
```

Comment:
that looks incorrect: your texture coordinates are going to be interpolated from the full-screen vertices, which is not desired